Libs/MRML/Core/vtkMRMLTransformDisplayNode.h

```diff
@@ -49,6 +49,7 @@ class VTK_MRML_EXPORT vtkMRMLTransformDisplayNode : public vtkMRMLDisplayNode
     VIS_MODE_GLYPH,
     VIS_MODE_GRID,
     VIS_MODE_CONTOUR,
+    VIS_MODE_INTERACTIVE,
```

Comment:
The reasons I ended up adding it as a visualization mode (contrary to what we decided at NA-MIC week) are:
 - I wasn't sure we should support both a visualization and the interaction at the same time. 
 - The interaction needs to rely on the Visibility and SliceIntersectionVisibility flags too. As of now, in the interface you cannot unselect a visualization mode (no visualization mode =VIS_MODE_NONE). So you wouldn't be able to have interaction without a visualization mode. 

So to make it a separate variable I think we would need to:
 - Have interaction and a visualization mode at the same time
 - Add a visualization mode VIS_MODE_NONE (or another better/appropriate name) to be able to only have the interaction on.
What do you think ?